utils.py

```diff
@@ -6,5 +10,98 @@ def filter_country_sort_year(df, country):
     :return: DataFrame
     """
     df = df.loc[df["country_name"] == country]
-    df = df.sort_values(["year"])
+
+    df = df[df.year >= START_YEAR]
+    df = df.sort_values(["year"], ascending=False)
     return df
+
+
+import math
+
+millnames = ["", " K", " M", " B", " T"]
+
+
+def millify(n):
+    n = float(n)
+    millidx = max(
+        0,
+        min(
+            len(millnames) - 1, int(math.floor(0 if n == 0 else math.log10(abs(n)) / 3))
+        ),
+    )
+
+    return "{:.2f}{}".format(n / 10 ** (3 * millidx), millnames[millidx])
```

Comment:
FYI @bhupatiraju 
Let's explicilty format the values for hover and narrative texts. The current formatting (default formatting by plotly) is not the best. (too many digits, I think 2 significance level or 3 is enough) 

**_Current formatting_**
![image](https://github.com/user-attachments/assets/aaa999fd-5da0-4096-846b-6b230834b25f)


**_New explicit formatting_**
![image](https://github.com/user-attachments/assets/6e7cfeeb-5beb-43a6-a809-d80337f4afd3)

Reply:
@yukinko-iwasaki This formatting is nice. I thought the default Plotly was ugly too. I'll make the edit in the subnational plots branch as well before merging it